resolver/resolver.cc

```diff
@@ -3560,6 +3560,7 @@ class ResolveSignaturesWalk {
                 if (typeSpec.type) {
                     auto name = ctx.state.freshNameUnique(core::UniqueNameKind::TypeVarName, typeSpec.name, 1);
                     auto sym = ctx.state.enterTypeArgument(typeSpec.loc, method, name, core::Variance::CoVariant);
+                    sym.data(ctx)->addLoc(ctx, typeSpec.loc);
```

Comment:
This is likely another case where we can get away with pushing the `addLoc` into the call to `enterTypeArgument`.